src/pages/Setting/components/index.tsx

```diff
@@ -0,0 +1,67 @@
+import { Link } from 'react-router-dom';
+// import { ReactComponent as ArrowLeft } from 'assets/svg/setting/arrow-left.svg';
+import { ReactComponent as ArrowRight } from 'assets/svg/setting/arrow-right.svg';
+import { ReactComponent as Move } from 'assets/svg/setting/movement.svg';
+import PreviousButton from 'components/PreviousButton/PreviousButton';
+import styles from './Setting.module.scss';
+
+export default function Setting(): JSX.Element {
+  return (
+    <div className={styles.container}>
+      <div className={styles.head}>
+
+        <div className={styles['head__left-arrow']}>
+          <PreviousButton />
+        </div>
+        <div className={styles.title}>설정</div>
+      </div>
+      <div className={styles.account}>
+        <div className={styles['sub-title']}>계정 관리</div>
+        <div className={styles.block}>
+          <div className={styles['account-text']}>아이디 변경</div>
+          <div className={styles['current-text']}>fgnsdbcxb</div>
```

Comment:
`useAuth()` hook을 만들어두었는데 이를 이용하면 쉽게 아이디에 접근할 수 있어요.
```ts
  const auth = useAuth();
```
```suggestion
          <div className={styles['current-text']}>{auth?.account}</div>
```

Reply:
+ 로그인되지 않았을 때는 auth가 null이라 나오지 않을텐데, 
비로그인 상태에서 해당 페이지에 접근하지 못하도록 막는 장치가 필요해보이네요

Reply:
수정했습니다!
77e42ea667a0b0ce83db590d1d6f4524b4f6eaa5